src/fvm_multicell.hpp

```diff
@@ -194,15 +240,38 @@ class fvm_multicell {
     }
 
 private:
-
     threshold_watcher threshold_watcher_;
 
     /// current time [ms]
-    value_type t_ = value_type{0};
+    value_type t_ = 0;
 
     /// resting potential (initial voltage condition)
     value_type resting_potential_ = -65;
 
+    /// final time in integration round [ms]
+    value_type tfinal_ = 0;
+
+    /// max time step for integration [ms]
```

Comment:
A quibble about naming: this could also be referred to as the `target_dt`, i.e. the time step size that we would like to take, events and `t_final` permitting. The comment might want to mention this.